critiquebrainz/frontend/templates/macros.html

```diff
@@ -93,8 +93,19 @@
   </div>
 {%- endmacro %}
 
+{% macro show_tags(tags) %}
+  {% if tags %}
+    {% for tag in tags[: -1] %}
+      <span><a href="{{"https://musicbrainz.org/tag/%s"|format(tag['name'])}}"> {{tag['name']}}, </a></span>
+    {% endfor %}
+    <span><a href="{{"https://musicbrainz.org/tag/%s"|format(tags[-1]['name'])}}"> {{tags[-1]['name']}} </a></span>
```

Comment:
I was trying to show all tags with a comma except the last tag. Should I try it some other way? (CSS, maybe)